README.rst

```diff
@@ -43,7 +43,7 @@ Tahoe is all about subdomains, so please add the following entries to your ``/et
 
     127.0.0.1 edx.devstack.lms  # Needed for devstack nascence
     127.0.0.1 red.localhost  # Your default site
-    127.0.0.1 blue.localhost green.localhost  # Create as many of those as you wish
+    127.0.0.1 blue.devstack.tahoe green.devstack.tahoe  # Create as many of those as you wish
 
 Where's my Site?
```

Comment:
Ahmed please modify that section to note that `create_devstack_site` is needed. Something like the following:

```diff
-Now you have a site of your own:
+To create a new site, please run the following commands: ... , ...
+
+Then you'll have a site of your own:
```